app/views/cases/offender_sar/_data_request_request_type.html.slim

```diff
@@ -0,0 +1,20 @@
+                = t :"cases.data_requests.index.request_type.#{data_request.request_type}"
+                - if data_request.other? && data_request.request_type_note.present?
+                  = ": #{data_request.request_type_note}"
+                - if data_request.request_dates_either_present?
+                  span.data-detail<
+                    br
+                    - if data_request.date_from.present?
```

Comment:
Just an idea - could we make these methods private and then just have a public wrapper method that outputs the right thing - then we don't have to repeat the spans? 🤔